src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -45,6 +45,22 @@
 logger = logging.getLogger(__name__)
 
 
+def _escape_version(version: str) -> str:
```

Comment:
This function name needs to be descriptive. Something like:
```suggestion
def _escape_project_version_for_markup_source(version: str) -> str:
```